cypher-shell/src/main/java/org/neo4j/shell/prettyprint/PrettyPrinter.java

```diff
@@ -14,25 +14,22 @@
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
 /**
  * Print the result from neo4j in a intelligible fashion.
  */
 public class PrettyPrinter {
     private static final String COMMA_SEPARATOR = ", ";
-
     private static final String COLON_SEPARATOR = ": ";
     private static final String COLON = ":";
     private static final String SPACE = " ";
-    public static final String BACKTICK = "`";
-    private static final Pattern ALPHA_NUMERIC = Pattern.compile("^[\\p{L}_][\\p{L}0-9_]*");
-    private StatisticsCollector statisticsCollector;
+    private final CypherVariablesFormatter cypherVariablesFormatter;
+    private final StatisticsCollector statisticsCollector;
 
     public PrettyPrinter(@Nonnull Format format) {
         this.statisticsCollector = new StatisticsCollector(format);
+        this.cypherVariablesFormatter = new CypherVariablesFormatter();
```

Comment:
If the method was static (no reason not to) then you would not have to keep an instance of the class around.